conans/client/build/autotools_environment.py

```diff
@@ -95,7 +96,7 @@ def _get_host_build_target_flags(self):
         return build, host, None
 
     def configure(self, configure_dir=None, args=None, build=None, host=None, target=None,
-                  pkg_config_paths=None, vars=None):
+                  pkg_config_paths=None, vars=None, default_install_dirs=True):
```

Comment:
``default_install_dirs`` > ``use_default_install_dirs``, I think it is more explicit if it is a True/False parameter.

Reply:
Thought about it but I think it is too long, however the boolean type could be more easily inferred